src/content/docs/contributing/modules.md

```diff
@@ -450,30 +431,43 @@ assertAll(
           )
 ```
 
-- Create a new `tags.yml` in the `modules/nf-core/<tool>/<subtool>/tests/` folder and add only the corresponding module tag from `tests/config/pytest_modules.yml`
+:::tip{title="pytest vs. nf-test assertions"}
 
-```yaml
-<tool>/<subtool>:
-  - modules/nf-core/<tool>/<subtool>/**
-```
+| pytest     | nf-test                                                                                       | description                                                                           |
+| ---------- | --------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| `md5sum`   | `assert snapshot(path(process.out.npl.get(0).get(1))).match(){:groovy}`                       | extract the file of a meta tuple from the npl channel, and check the snapshot matches |
+| `contains` | `assert path(process.out.npo.get(0).get(1)).getText().contains("Nonpareil version"){:groovy}` | check to see if it contains a given string                                            |
+| `path`     | `assert snapshot(file(process.out.log.get(0).get(1)).name).match("log"){:groovy}`             | check to see if the filename is "log"                                                 |
 
-- Run the test to create a snapshot of your module test. This will create a `.nf.test.snap` file
+:::
+
+- Run the test to create a snapshot of your module test. This will create a `main.nf.test.snap` file
 
 ```bash
 nf-core modules test <tool>/<subtool>
 ```
 
-:::note
-Remove the corresponding tags from `tests/config/pytest_modules.yml` so that py-tests for the module will be skipped on github CI
-:::
+- Remove the corresponding tags from `tests/config/pytest_modules.yml` so that py-tests for the module will be skipped on github CI.
```

Comment:
suggest moving this step to before L 573 where we ask to create `tags.yml` as it will be easy to copy from pytest yml first and then delete it

Reply:
For unchained modules, people don't need to touch the `tags.yml` because the create command populates it already with the correct data.